Cartfile

```diff
@@ -1,2 +1,2 @@
-binary "https://assets.braintreegateway.com/mobile/ios/carthage-frameworks/cardinal-mobile/CardinalMobile.json" == 2.2.5
-binary "https://assets.braintreegateway.com/mobile/ios/carthage-frameworks/pp-risk-magnes/PPRiskMagnes.json" == 5.0.1
+binary "https://assets.braintreegateway.com/mobile/ios/carthage-frameworks/cardinal-mobile/CardinalMobile.json" == 2.2.5-xcf
+binary "https://assets.braintreegateway.com/mobile/ios/carthage-frameworks/pp-risk-magnes/PPRiskMagnes.json" == 5.2.0
```

Comment:
For Magnes, we updated to the latest version and added both a framework and xcframework.

For Cardinal, we don't have a newer version, so we added `2.2.5-xcf` as a "new" version for which we have both a framework and an xcframework. This is just to make sure we don't break anything in previous versions of the Braintree SDK.